### PR TITLE
Updated frontend

### LINKED ISSUE #29: New website design
unknown

### PR DESCRIPTION
**Pull request by YKo20010**
image

### CODE REVIEW

**Review by chrisnguyn — APPROVED** [review]
wow what a great programmer

File: web/static/js/googlemap.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            if (place.geometry.viewport) {
+                // Only geocodes have viewport.
+                bounds.union(place.geometry.viewport);
+            } else {
+                bounds.extend(place.geometry.location);
+            }
+        });
+        map.fitBounds(bounds);
+    });
+}
```
Comment (kx-chen) [inline web/static/js/googlemap.js:72]: Add new line.

File: web/static/js/sidebar.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+}
+
+function addHeader(parent, text) {
+    return parent.append("div")
+        .attr("class", "header")
+        .style("width", width - padding.horizontal * 2)
+        .style("word-wrap", "break-word")
+        .style("color", color.ink)
+        .text(text)
+}
```
Comment (kx-chen) [inline web/static/js/sidebar.js:16]: Add new line.

File: web/static/styles/googlemap.css [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    color: #fff;
+    background-color: #4d90fe;
+    font-size: 25px;
+    font-weight: 500;
+    padding: 6px 12px;
+}
+
+#target {
+    width: 345px;
+}
```
Comment (kx-chen) [inline web/static/styles/googlemap.css:84]: Add new line.

File: web/static/styles/style.css [hunk 4]
```diff
@@ -1,185 +1,22 @@
...
-.cover2 {
-  position: fixed;
-  display: block;
-  height: 100vh;
-}
+.title {
+    font-family: 'Abel', sans-serif;
+    font-size: 48px;
+    text-align: center;
+}
```
Comment (kx-chen) [inline web/static/styles/style.css:22]: Add new line.

File: web/templates/index.html [hunk 5]
```diff
@@ -1,180 +1,171 @@
...
+            wordPhraseText.text(text)
+        }
+
+        updatePlusCode("6PH57VP3+CW7")
+        updateWordPhrase("between.west.north.front.body.board")
+    </script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_KEY_HERE&libraries=places&callback=initAutocomplete " async defer></script>
+</body>
+
+</html>
```
Comment (kx-chen) [inline web/templates/index.html:171]: Add new line.

**Review by kx-chen — APPROVED** [review]

**Pull request merged**